code/datums/religions.dm

```diff
@@ -1036,3 +1039,40 @@
 	H.h_style = "Bald"
 	H.f_style = "Shaved"
 	H.update_hair()
+
+/datum/religion/guns
+	name = "Murdercube"
+	deity_name = "Gun Jesus"
+	bible_name = "The Dokument"
+	male_adept = "Kommando"
+	female_adept = "Kommando"
+	keys = list("murdercube","murder/k/ube","forgotten weapons", "gun", "guns", "ammo", "trigger discipline", "ave nex alea")
+	convert_method = "performing a ritual with a gun. The convert needs to be in good health and unafraid of being shot."
+
+/datum/religion/guns/equip_chaplain(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/weapon/gun/energy/laser/practice)
+	H.equip_or_collect(new /obj/item/clothing/under/syndicate, slot_w_uniform)
+	H.equip_or_collect(new /obj/item/clothing/shoes/jackboots, slot_shoes)
+
+/datum/religion/guns/convertCeremony(var/mob/living/preacher, var/mob/living/subject)
+	var/held_gun = preacher.find_held_item_by_type(/obj/item/weapon/gun)
+
+	if (!held_gun)
+		to_chat(preacher, "<span class='warning'>You need to hold a gun to begin the conversion.</span>")
+		return FALSE
+
+	if(!convertCheck(subject))
+		subject.visible_message("<span class='warning'>\The [subject] refuses conversion.</span>")
+		return FALSE
+
+	var/obj/item/weapon/gun/G = preacher.held_items[held_gun]
+
+	sleep(0.1 SECONDS)
+	G.Fire(subject,preacher,0,0,1)
```

Comment:
```suggestion
	if(G.can_shoot())
		G.Fire(subject,preacher,0,0,1)
	else
		preacher.visible_message("<span class='warning'>*click*</span>")
		playsound(preacher, 'sound/weapons/empty.ogg', 100, 1)
```

Reply:
no pussy shit with unloaded guns allowed

Reply:
If you merge it right after this, I'll do it.